Makefile

```diff
@@ -119,10 +120,10 @@ bundle:
 .PHONY: packagemanifests
 packagemanifests:
 	$(OPERATOR_SDK) generate kustomize manifests -q --input-dir $(OLM_MANIFESTS) --output-dir $(OLM_MANIFESTS) --apis-dir pkg/apis
-	$(KUSTOMIZE) build $(OLM_MANIFESTS) | sed "s|intel-deviceplugin-operator:devel|intel-deviceplugin-operator:$(OPERATOR_VERSION)|" | $(OPERATOR_SDK) generate packagemanifests -q --kustomize-dir $(OLM_MANIFESTS) --version $(OPERATOR_VERSION) --from-version $(OPERATOR_PREVIOUS_VERSION) $(BUNDLE_METADATA_OPTS)
-	# Remove unneeded resources
-	rm packagemanifests/$(OPERATOR_VERSION)/*service.yaml
-	rm packagemanifests/$(OPERATOR_VERSION)/*clusterrole.yaml
+	$(KUSTOMIZE) build $(OLM_MANIFESTS) | sed "s|intel-deviceplugin-operator:devel|intel-deviceplugin-operator:$(OPERATOR_VERSION)|" | $(OPERATOR_SDK) generate packagemanifests -q --kustomize-dir $(OLM_MANIFESTS) --version $(OPERATOR_VERSION) --from-version $(OPERATOR_PREVIOUS_VERSION) $(BUNDLE_METADATA_OPTS) --output-dir $(PACKAGEMANIFESTS_DIR)
+	# Remove unneeded resourcess
```

Comment:
```suggestion
	# Remove unneeded resources
```

Reply:
Idk why I always make such "unneeded" typos.. Sorry, let me be more careful!